internal/tiltfile/links.go

```diff
@@ -0,0 +1,93 @@
+package tiltfile
+
+import (
+	"fmt"
+
+	"go.starlark.net/starlark"
+
+	"github.com/tilt-dev/tilt/pkg/model"
+)
+
+func convertLinks(val starlark.Value) ([]model.Link, error) {
```

Comment:
can you make this implement the Unpack interface? here's a good example
https://github.com/tilt-dev/tilt/blob/master/internal/tiltfile/value/duration.go#L21